web/kweb.cfg

```diff
@@ -1,8 +1,9 @@
-:80 {
+*:80 {
     redir / https://{host}{uri}
 }
 
-*, :443 {
+# kweb >=0.7.0 needs set the explicit hostname (domain needs to be detectable by sni)
+{%FQDN%}, *, *:443 {
```

Comment:
It should not be required to add the FQDN here expliclty.

Reply:
after investigation we came to the conclusion that for lets encrypt cert generation to work the fqdn indeed needs to be included